backend/inspirehep/records/marshmallow/literature/es.py

```diff
@@ -44,6 +47,25 @@ class LiteratureElasticSearchSchema(ElasticSearchBaseSchema, LiteratureRawSchema
     def get_ui_display(self, record):
         return json.dumps(LiteratureDetailSchema().dump(record).data)
 
+    def get_latex_us_display(self, record):
+        from inspirehep.records.serializers.latex import latex_US
```

Comment:
:cry: I guess you get circular import otherwise?

Reply:
:( yes

Reply:
no way to rearrange things to avoid that?

Reply:
Couldn't find a way, things in here are way too twisted together